Project/Source/Components/UI/ComponentImage.cpp

```diff
@@ -233,6 +245,14 @@ void ComponentImage::SetAlphaTransparency(bool alphaTransparency_) {
 	alphaTransparency = alphaTransparency_;
 }
 
+TESSERACT_ENGINE_API float2 ComponentImage::GetTextureOffset() const {
+	return textureOffset;
+}
+
+TESSERACT_ENGINE_API void ComponentImage::SetTextureOffset(float2 textureOffset_) {
```

Comment:
You don't really need the TESSERACT_ENGINE_API notation here (just in the .h). I suppose this has been tested in a Gameplay script and works correctly with the notation.

Reply:
Correct, this isn't necessary

Reply:
Copy that